### ISSUE TITLE
US Tribal data rears its uncertain head again!

### ISSUE THREAD

**Issue by aaronmberger** [cgrandin/hake-assessment#562]
Sorry for the long issue here but I want to keep track of it.  Message from Dan Waldeck first, followed by my response to him after that.

[From Dan:
Hey Aaron,

First, a hearty thanks for the yeoman's effort in getting the assessment 
pulled together.

It probably makes little difference to current stock status, but I want 
to ensure the JTC is using an accurate amount for the tribal hake 
fishery.  From the SRG-draft assessment, it appears that the JTC is 
using 2,649 mt for tribal hake catch in 2018; that is, the catch tables 
(table a and table 2) note that "Tribal catches are included in the 
sector totals" (I presume shoreside is the only sector with tribal catch 
included) -- PacFIN shows SS at 129,180 mt and the assessment shows SS 
at 131,829 mt, which would seem to indicate tribal catch of 2,649 mt 
(i.e., the latter minus the former).

However, from the PFMC's GMT, my understanding is that 2018 tribal catch 
was 5,706 mt.

I cc Jessi [WDFW] for her thoughts too.]

My Response:
[Hi Dan,

Thanks for reaching out on the tribal catch.  I've double checked our database extraction and our numbers are correct (given our standard process, but maybe that needs revision).  The discrepancy that you point out arises because of the following (the best I can tell):

1. The JTC extracts the US shoreside data used in the assessment from PacFIN, which for the WA component of that includes the aggregation of both WA commercial catches (Participant_code 'C' in PacFIN) and WA Tribal catches (Participant_code 'I' in PacFIN).

2. We typically do not separate out codes 'C' and 'I' in our extractions, but rather just take the lump sum.

3. We do however look at catch by fleet, which separates out Fleet_code 'TI' for tribal (among others), but that has recently been 0 mt (and is again this year), presumably because tribal catch is being ticketed as 'tribal participant' in WA fishery, not as a separate FLEET (odd to me, but I presume there are good reasons for this coding).

4. We did receive word from Jessi D. (and others) that the tribal catch was 5,706.4 mt for 2018 but that there was still lingering errors with some fish tickets such that it may not all make it into PacFIN until after the assessment due date.

5. The difference that you lay out of nearly ~3,000 mt is what I assume was not entered into PacFIN before our final data extraction.

6. As done in the past, the JTC used data in PacFIN as the best available (QA/QC'ed) source of information and that any differences due to tribal tickets not making it in on time would be small.  

From my perspective, there are a couple of things at play here.  First, we almost always have additional catch added to the previous years total the following year because updates to catch totals occur in PacFIN after our data deadline.  We see the effect of these on the stock assessment each year through our bridging analysis (I know your familiar with these).  Of course, we should still strive to include the most up to date information we can in any given assessment year, but at the same time maintain repeatability.  As mentioned, I think this year we were under such a time crunch that we assumed that any lingering tribal fish tickets not in PacFIN would be trivial and didn't have time to double check what the total missing catch (relative to the 5,706.4) actually was (turns out to be ~3,000 mt not in PacFIN at the time of our extraction).  

So what to do?  We lose some degree of 'repeatability' if we have to annually track down what has and what has not been uploaded to PacFIN and at what QA/QC stage those data not in PacFIN are at.  At the same time, if we get numbers from the GMT (like we did this year), it seems reasonable to use those totals in the assessment.  The latter requires that additional step of resolving how much of the GMT total is in PacFIN and how much is not (so we get the final correct number).

For now, what we can do is run a model with the additional tribal catch provided by the GMT.  As you say, it will not make much of any difference in stock status or related management quantities.  Perhaps more importantly, it would be good to have a discussion at the SRG (or perhaps even better suited for the JMC) about the process for getting hake tribal data so that we can collectively agree on a best process that makes sense and is, for the most part, repeatable from year to year.  

Sorry for the long winded response.  And thanks again for pointing this out and bringing it to our attention.  And yes, Jessi may have additional welcomed insight here.]

My thoughts to you guys on this:
In general, I don't think it is really appropriate for the JTC to determine what catch data is appropriately QA/QC'd for use in stock assessment, thus we have used what's in PacFIN in the past and any lingering fish tickets not entered get incorporated the following year and formally examined in bridging runs.  However, if the GMT (Groundfish Management Team for the US Pacific Fisheries Management Council) provides a number in a timely fashion (on or before first week in January), I'm sure we can figure out a way to add those catches in to the assessment.  The challenge is that of the total we get from the GMT, some or part or all of it is in PacFIN (like this year), so we are left to figure out how much.  

Any thoughts on this (?), as it would be good to have a JTC collective opinion as I'm sure this will (again) come up at the SRG (though it probably is a question to be addressed by the JMC).

For now, I'll run a model with the additional tribal catch and pull out key comparisons (stock status) with our base model, just so we have something prepared for the meeting.  I don't think we need to have something in our presentations at this point, but rather just prepared to speak about it if/when asked.

**Comment by kellijohnson** [comment]
I will explore the database a little bit to see what is and what isn't in there currently and what was and wasn't in there when we did our data extraction.

I think that attempting to get a number from the GMT is putting a bandaid on the situation and we need to advocate **strongly** for proper entry into PacFIN by the Jan 04 (or whatever date is appropriate) deadline. If we accommodate then they have no incentive to deliver on time. With that, I think your idea of running an additional model but not including it in anything is good to have.

**Comment by cgrandin** [comment]
I agree that there is only so much the JTC can do, and that it sounds like the QC on these data aren't good. We could re-run all models and reproduce the assessment with these values added but then are we just chasing noise, i.e. how good are these new catch numbers and will they be changed again next year?

Once course of action is to do nothing and then perhaps something will happen on the data QC end to help us in the future. As it stands, we tend to do a lot of scrambling around re-running models when these things are discovered post-assessment. Running on MLE model to show the difference to the SRG wouldn't hurt and it sounds like that's what you are doing.

**Comment by aaronmberger** [comment]
Yes, just to have the MLE version done to simply say it doesn't change management quantities.  My guess is that it will be within the rounding error on relative spawning biomass estimate.

I think WA was having trouble with PacFIN this year because they also had ~240 aged fish not make it in there in time for our assessment either (and remember we did a data draw in late January this year due to shutdown, not our normal first week in January, so there was even 'extra' time this year).

**Comment by aaronmberger** [comment]
Still confirming what catch number to use with this 'just for our information' trial MLE run, but using Waldeck's number (which is likely to be real close and as good as any for this exploration) there is almost no difference in estimates (rounding or otherwise):

Model      B_ratio_2019      SSB_2019      Rel_F_intesity_2018     Total 2018 catch

base             0.486255          1.67E+06          0.933334                410,443
add_tribal     0.485456          1.67E+06          0.936218                413,500

**Comment by kellijohnson** [comment]
Summary - PacFIN hasn't been updated regarding Shorebased landings since we did our data extraction on 2019-01-29. I added a column to specifically find out which Washington landings were tribal. PacFIN is missing (5706 - 2423.403 = ) 3282.6 t of tribal landings. We have no way of knowing if these are just not properly attributed to the tribal fishery within the shorebased landings or if they are not in the database. AMB ran an additional model to determine what happens if the later case is true and next year the database will be updated to include these catches from the tribal sector.

PacFIN 2019-02-14 
 - 2423.403 t of landings attributed to the tribal fishery within the shorebased sector.
 - 45235.73 t of landings attributed to the LE Washington fishery within the shorebased sector.
 - 131829.20 t of landings attributed to Shorebased sector.

PacFIN 2019-01-29 (i.e., date of data extraction)
 - 47659.13 t of landings attributed to Washington in the shorebased sector.
 - 131829.20 t of landings attributed to Shorebased sector.

Future - The US data extraction will include information specifically regarding how much of the shorebased sector landings came from the tribal sector according to PacFIN. This number can then be compared to what Jessi supplies and we can let them know how many metric tons are missing early in the game and they can work to make sure they are uploaded by our deadline of the first week of January.
